<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TvShowWidgetSeason</class>
 <widget class="QWidget" name="TvShowWidgetSeason">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>732</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="buttonRevert">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string>Episode has changed. Click to revert changes.</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../data/MediaElch.qrc">
             <normaloff>:/img/arrow_circle_left.png</normaloff>:/img/arrow_circle_left.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="title">
           <property name="font">
            <font>
             <family>Helvetica Neue</family>
             <pointsize>22</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Title</string>
           </property>
           <property name="indent">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="missingLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Season missing</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="labelThumb">
          <property name="text">
           <string>Thumb</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ClosableImage" name="thumb">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>102</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>102</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Click to Change</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../../../data/MediaElch.qrc">:/img/placeholders/thumb.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="clickable" stdset="0">
           <bool>true</bool>
          </property>
          <property name="myFixedHeight" stdset="0">
           <number>102</number>
          </property>
          <property name="dropShadow" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="labelPoster">
          <property name="text">
           <string>Poster</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ClosableImage" name="poster">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>290</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>290</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Click to Change</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../../../data/MediaElch.qrc">:/img/placeholders/poster.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="clickable" stdset="0">
           <bool>true</bool>
          </property>
          <property name="myFixedHeight" stdset="0">
           <number>290</number>
          </property>
          <property name="dropShadow" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelFanart">
          <property name="text">
           <string>Fanart</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ClosableImage" name="backdrop">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>120</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>120</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Click to Change</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../../../data/MediaElch.qrc">:/img/placeholders/fanart.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="clickable" stdset="0">
           <bool>true</bool>
          </property>
          <property name="myFixedHeight" stdset="0">
           <number>120</number>
          </property>
          <property name="dropShadow" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelBanner">
          <property name="text">
           <string>Banner</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ClosableImage" name="banner">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>53</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>53</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Click to change</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../../../data/MediaElch.qrc">:/img/placeholders/banner.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="clickable" stdset="0">
           <bool>true</bool>
          </property>
          <property name="myFixedHeight" stdset="0">
           <number>53</number>
          </property>
          <property name="dropShadow" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClosableImage</class>
   <extends>QLabel</extends>
   <header>ui/small_widgets/ClosableImage.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../data/MediaElch.qrc"/>
 </resources>
 <connections/>
</ui>
